www/front_src/src/Resources/index.test.tsx

```diff
@@ -674,9 +696,147 @@ describe(Resources, () => {
 
     fireEvent.click(getByText(labelAcknowledge));
 
-    await wait(() => expect(mockedAxios.get).toHaveBeenCalled());
+    await waitFor(() => expect(mockedAxios.get).toHaveBeenCalled());
 
     expect(getByText(labelAcknowledgedByAdmin)).toBeInTheDocument();
     expect(queryByText(labelAcknowledgeServices)).toBeNull();
   });
+
+  /**
+   * Downtime dialog
+   */
+
+  const selectAllResourcesAndPrepareToSetDowntime = async ({
+    getByLabelText,
+    getByText,
+  }): Promise<void> => {
+    await selectAllResources({ getByLabelText });
+
+    fireEvent.click(getByText(labelDowntime));
+
+    await waitFor(() => expect(mockedAxios.get).toHaveBeenCalled());
+  };
+
+  const labelDowntimeByAdmin = `${labelDowntimeBy} admin`;
+
+  it('cannot send a downtime request when Downtime action is clicked and comment is empty', async () => {
+    const { getByLabelText, getByText, getAllByText } = render(<Resources />);
+
+    await selectAllResourcesAndPrepareToSetDowntime({
+      getByLabelText,
+      getByText,
+    });
+
+    fireEvent.change(getByText(labelDowntimeByAdmin), {
+      target: { value: '' },
+    });
+
+    await waitFor(() =>
+      expect(last(getAllByText(labelSetDowntime)).parentElement).toBeDisabled(),
+    );
+  });
+
+  it('cannot send a downtime request when Downtime action is clicked, flexible and duration is empty', async () => {
```

Comment:
What is flexible?

Reply:
the downtime, according to the documentation : https://documentation.centreon.com/docs/centreon/en/latest/api/clapi/objects/realtime_downtimes.html?highlight=downtime

Reply:
I've update the title to avoid confusion